src/test/skript/tests/regressions/6707-spawn-fish-types.sk

```diff
@@ -0,0 +1,26 @@
+test "spawn fish types":
+	spawn a cod at spawn of world "world":
+		set {_e} to entity
+	assert {_e} is a cod with "failed to spawn a cod"
+	delete entity within {_e}
+
+	spawn a salmon at spawn of world "world":
+		set {_e} to entity
+	assert {_e} is a salmon with "failed to spawn a salmon"
+	delete entity within {_e}
+
+	spawn a puffer fish at spawn of world "world":
+		set {_e} to entity
+	assert {_e} is a puffer fish with "failed to spawn a puffer fish"
+	delete entity within {_e}
+
+	spawn a tropical fish at spawn of world "world":
+		set {_e} to entity
+	assert {_e} is a tropical fish with "failed to spawn a tropical fish"
+	delete entity within {_e}
+
+	parse if running minecraft "1.19":
+		spawn a tadpole at spawn of world "world":
+			set {_e} to entity
+		assert {_e} is a tadpole with "failed to spawn a tadpole"
+		delete entity within {_e}
```

Comment:
should this be parse or `test "blah" if running minecraft 1.19`? or whatever the syntax was for version testing

Reply:
That'd make it a different test, and I believe I'd still need to gate it behind a parse if. `tadpole` just doesn't exist prior to 1.19.

Reply:
taking ExprFreezeTicks test as an example `test "freeze time" when running minecraft "1.18":` pretty sure you don't